Arrowgene.Ddon.Shared/AssetReader/EnemySpawnAssetDeserializer.cs

```diff
@@ -97,6 +97,7 @@ public EnemySpawnAsset ReadPath(string path)
                     IsAreaBoss = row[enemySchemaIndexes["IsAreaBoss"]].GetBoolean(),
                     BloodOrbs = row[enemySchemaIndexes["BloodOrbs"]].GetUInt32(),
                     HighOrbs = row[enemySchemaIndexes["HighOrbs"]].GetUInt32(),
+                    SpawnTime = row[enemySchemaIndexes["SpawnTime"]].GetString(),
```

Comment:
For compatibility with older files that don't have spawn time properties, the server should be able to read jsons with no spawn time and give them a sane default value (all day). Check line 52 for an example